README.md

```diff
@@ -470,6 +471,32 @@ second subscriber: 3
 second subscriber: 4
 ```
 
+### prefix(duration:)
+
+A time-based overload on `Publisher.prefix` that accepts value and completions events from an upstream publisher for a specified `duration` of time (in seconds). After expiring, the operator will emit a `.finished` event and ignore all future output. Configuration options are also exposed for the underlying timer, if needed.
```

Comment:
This explanation is a bit verbose and has a lot of impementation details, I think it would be better to just mention it's an overload of `prefix` that republishes values for the provided duration, and then completes